src/components/dictionaryConcepts/components/DescriptionTable.jsx

```diff
@@ -31,7 +31,7 @@ const DescriptionTable = props => (
 );
 
 DescriptionTable.propTypes = {
-  description: PropTypes.string.isRequired,
+  description: PropTypes.array.isRequired,
```

Comment:
Arrays are best defined in the plural form. `description` best suits an object or string. Could you please create a ticket to address this?

Reply:
Okay. Will do that.